### DISCUSSION TITLE
Finally, Kodi support

### DISCUSSION THREAD

**Discussion by Skaytacium** [Syncplay/syncplay#560] [Show and tell]
I've started support for Kodi, but its basically like making my own Syncplay client. The repo is https://github.com/Skaytacium/synko.

**Comment by Et0h** [comment]
I've never used Kodi but I know that people have asked for Kodi support over the years (e.g. #350) so thanks for your work on this. @tari did some work on XBMC support back in 2014 but that never got to the release stage.

In addition to looking at https://syncplay.pl/about/protocol/ and the source code for the official Syncplay clients/erver, other third-party implementations of Syncplay's protocol might also be of interest/assistance. See:
* Syncplay Android by @Reddnek which can be found at https://github.com/Reddnek/syncplay-android - I've done some posts on the issues  to support Syncplay Android showing how the Syncplay protocol/client is supposed to work, e.g. https://github.com/Reddnek/syncplay-android/issues/6 and https://github.com/Reddnek/syncplay-android/issues/17 and https://github.com/Syncplay/syncplay/discussions/542
* Syncplay Web by @mo3rfan at https://github.com/mo3rfan/syncplay-web
* Syncplayer for Android by @mo3rfan at https://github.com/mo3rfan/syncplayer
* Syncplay Java by @mo3rfan at https://github.com/mo3rfan/syncplay-java
* Syncplay.net by @arkangel-dev at https://github.com/arkangel-dev/SyncPlay.NET

**Comment by Skaytacium** [comment]
Those are quite interesting respositories. Since I have finished the project and only plan on maintaining it from now on, ironing out any bugs  reported by the community, I don't think I'll be adding any more features until they are explicitly requested for, but these will definitely be helpful.
